configuration/fisBomToFabric8MavenPlugin.fuse7.properties

```diff
@@ -8,4 +8,5 @@
 3.0.11.fuse-740029-redhat-00002=3.5.33.fuse-740029-redhat-00001
 3.0.11.fuse-750035-redhat-00001=3.5.33.fuse-750033-redhat-00001
 3.0.11.fuse-760025-redhat-00001=3.5.33.fuse-760025-redhat-00001
-3.0.11.fuse-770009-redhat-00002=3.5.42.fuse-770010-redhat-00001
\ No newline at end of file
+3.0.11.fuse-770009-redhat-00002=3.5.42.fuse-770010-redhat-00001
+3.0.12=4.3.0.fuse-780012-redhat-00001
```

Comment:
I know you mentioned that Gregorz told you it is normal in private chat to have a community version. Can you put the reason of no more using a productized version here please?

Reply:
@apupier the 3.0.11.x productized versions were created/built from `github/jboss-fuse/fabric8` fork. We no longer maintain this fork (at least in 7.x), because the upstream version, which was supposed to be merged into the fork regularly is no longer maintained. The upstream `github/fabric8io/fabric8` is split into many separate repositories and we've simply decided to use upstream version of "fabric8" - especially because we've switched to separate releases of `kubernetes-model` and `kubernetes-client`.
The OSGi parts of "fabric8v2" were moved to `github/jboss-fuse/fuse-karaf` project and there's really not much left needed from `fabric8v2`.

I'm not the best person to talk about "fabric8v2", I'd ask Nicola Ferraro, Luca Burgazzoli, Jon Anstey or Tom Cunningham. I've never dealt with kubernetes/openshift deployment using "fabric8v2"... Please also check `git blame` of all the quickstarts from https://github.com/fabric8-quickstarts/ project to check who's the expert ;)